workspaces/liferay-sample-workspace/client-extensions/liferay-sample-custom-element-with-third-party-cookie/assets/index.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+(function () {
+	'use strict';
+
+	class CustomElementThirdPartyCookie extends HTMLElement {
+		constructor() {
+			super();
+
+			const div = document.createElement('div')
+			
+			const link = document.createElement('link');
+			link.setAttribute('data-href', 'https://liferay-3rd-party-cookie.vercel.app/api/set-cookie.css?type=link');
+			link.setAttribute('rel', 'stylesheet');
+			link.setAttribute('type', 'text/css');
+			link.setAttribute('data-third-party-cookie', 'CONSENT_TYPE_PERSONALIZATION');
```

Comment:
:clap: nice! 

By doing this script generation on the fly we are proving that enforcement mechanism need to run at browser side.

We can refine and do this server side in some specific cases as a complementary mechanism.